addons/mail/static/src/components/partner_im_status_icon/partner_im_status_icon.js

```diff
@@ -29,6 +29,7 @@ export class PartnerImStatusIcon extends Component {
         if (!this.props.hasOpenChat) {
             return;
         }
+        ev.stopPropagation();
```

Comment:
Why this `stopPropagation`? Which part of the code actually intercepted the click and it's undesirable?